pkg/params/run.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	PACConfigmapName        = "pipelines-as-code"
 	StartingPipelineRunText = `Starting Pipelinerun <b>%s</b> in namespace
-  <b>%s</b><br><br>You can follow the execution on the [OpenShift console](%s) pipelinerun viewer or via
+  <b>%s</b><br><br>You can follow the execution on the [%s](%s) PipelineRun viewer or via
```

Comment:
what is the value replaces %s here :thinking:

Reply:
just earlier int he commit https://github.com/openshift-pipelines/pipelines-as-code/pull/1057/files#diff-7d95c98719a2b5fe21a2104db3b95cda3425ab76566525da1437521470481c79R125

Reply:
OpenShift Console for openshift, Tekton Dashboard for kubernetes